spec/pathwatcher-spec.coffee

```diff
@@ -114,6 +114,21 @@ describe 'PathWatcher', ->
           done()
       fs.writeFileSync(tempFile, 'changed')
 
+  describe 'when watching a file that does not exist', ->
+    it 'throws an error with a code', (done) ->
```

Comment:
Aha. I was only reading the general Jasmine docs at http://jasmine.github.io/, not realizing that [jasmine-node](https://github.com/mhevery/jasmine-node) adds an alternate syntax for async tests that activates based on the `length` of the test function. Got it.

Updated PR to not use `done`.